README.rst

```diff
@@ -85,3 +85,69 @@ testing with live email addresses.  It works like this:
     # Destroy the account once you're done with it.
     acct.clear()
     client.destroy_account(acct.email, "MySecretPassword")
+
+
+Using Firefox Account BrowserID with Requests
+=============================================
+
+You can use the ``FxABrowserIdAuth`` to build the BrowserId assertion:
```

Comment:
Suggest capitalizing as "BrowserID" uniformly throughout, which is mostly how we've styled it in other places.